src/custom/profiles/vc_sd_jwt/mod.rs

```diff
@@ -0,0 +1,34 @@
+pub mod authorization_detail;
+pub mod credential_configuration;
+pub mod credential_request;
+pub mod credential_response;
+
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+pub const FORMAT_IDENTIFIER: &str = "vc+sd-jwt";
```

Comment:
Only question is should we make this distinct so it is clearly a Spruce-defined format identifier, to avoid potential name clashes in the future? I don't think the identifier has to be in a particular format (the current one looks MIME type-y to me).